media-tv/mythtv/metadata.xml

```diff
@@ -16,7 +16,7 @@
 		<flag name="bluray">Use the system libbluray for BluRay support</flag>
 		<flag name="cec">Allows control of CEC enabled TVs via HDMI</flag>
 		<flag name="ceton">Ceton InfiniTV 4 a CableCARD-enabled tuner support</flag>
-		<flag name="crystalhd">Broadcom CrystalHD hardware support</flag>
+		<flag name="crystalhd">Broadcom CrystalHD hardware support needs libcrystalhd to build</flag>
```

Comment:
This **needs** to be removed. Its not functional. We dont serve dependencies that can be got from an overlay, or "if you happened to install this in 2010".

Reply:
Removed from both versions